src/serialization.rs

```diff
@@ -93,7 +94,7 @@ pub(super) fn serialize_delta_set(delta_set: DeltaSet, last_sync_timestamp: u32)
 		announcements: vec![],
 		updates: vec![],
 		full_update_defaults: Default::default(),
-		chain_hash: Default::default(),
+		chain_hash: BlockHash::all_zeros(),
```

Comment:
I'm not finding in this PR how you make sure the serialized delta set returns the mainnet chain hash

Reply:
sorry, I had forgotten and didn't look at the full code